### PR TITLE
feat: add auth page reponsiveness

### PR DESCRIPTION
**Pull request by juniorboos**
Relates to:
- #971

## Screenshots (if visual changes)
<table align="center" border="0" cellpadding="0" cellspacing="0">
  <thead>
    <tr>
      <td>
       Old
      </td>
      <td>
        New
      </td>
    </tr>
  </thead>
<tr>
      <td>
image
      </td>
      <td>
image
      </td>
    </tr>
<tr>
      <td>
image
      </td>
      <td>
image
      </td>
    </tr>
<tr>
      <td>
image
      </td>
      <td>
image
      </td>
    </tr>
</table>

## Proposed Changes
  - Changed the `ImageContainer` CSS to include `flex-grow: 1` instead of the `FormContainer`, so it can be dynamic depending on the viewport.
  - Replaced some of the hardcoded values with theme tokens.
  - Fixed unnecessary horizontal scroll on smaller screens.

**Note: I recommend running this locally to avoid unexpected behaviours**

### CODE REVIEW

File: frontend/src/components/layouts/AccessLayout/styles.tsx [hunk 1]
```diff
@@ -2,6 +2,49 @@ import { styled } from '@/styles/stitches/stitches.config';
...
+
+  maxWidth: '500px',
+  height: 'fit-content',
+
+  display: 'flex',
+  flexDirection: 'column',
+
+  backgroundColor: '#ffffff',
+  borderRadius: '$12',
+});
```
Comment (StereoPT) [inline frontend/src/components/layouts/AccessLayout/styles.tsx:18]: This is an unused style.

**Review by CatiaBarroco-xgeeks — APPROVED** [review]
🎸

File: frontend/src/components/layouts/AccessLayout/styles.tsx [hunk 2]
```diff
@@ -2,6 +2,34 @@ import { styled } from '@/styles/stitches/stitches.config';
...
+
+  '@md': {
+    margin: '$48 $72 $24',
+    width: '$362',
+  },
+});
+
+const FormContainer = styled(Flex, {
+  height: '100%',
+  justifyContent: 'space-between',
```
Comment (StereoPT) [inline frontend/src/components/layouts/AccessLayout/styles.tsx:18]: The Flex Primitive has a `justifyContent` property, so I think it's better to just use Flex and add the css for the height

**Review by StereoPT — APPROVED** [review]
Everything else LGTM 🗝️

File: frontend/src/components/layouts/AccessLayout/styles.tsx (hunk 2 above)
Comment (juniorboos) [reply frontend/src/components/layouts/AccessLayout/styles.tsx]: Good to know, changed in a9f94b8!

**Review by StereoPT — APPROVED** [review]

**Pull request merged**